app/controllers/internal/application_controller.rb

```diff
@@ -2,6 +2,34 @@ class Internal::ApplicationController < ApplicationController
   before_action :authorize_admin
   after_action :verify_authorized
 
+  MENU_ITEMS = [
```

Comment:
i think it would be nice to either add a comment or tweak the const name here; at first glance, i was confused why this was here and it wasn't until i got to [the navbar](https://github.com/thepracticaldev/dev.to/pull/8161/files#diff-0b025b48805ca7b6093e4b356b4a723aR2) that i figured out what the deal was. especially would be useful i think because you don't usually see this kind of large frozen object in a controller

Reply:
another thought: can we put this in an instance variable that is then accessible by the view and can be at the bottom of the controller? thinking that this could be an option because it looks like this object is used solely for display purposes in the navbar...

Reply:
Good call - both make sense to me and it is confusing as is 😂 

I added a comment to clarify the use. I see what you mean about using an instance variable since it gets used in a view.

I'm thinking of sticking with a constant since the value is constant - we only need it initialized once and we can save ourselves the memory.

It also feels a little weird to put instance variables for a shared view in the application controller...maybe that part is just me 🙈 If it were a siloed view attached to a controller and an action, it would feel less weird. And since this is all in internal which is kinda, sorta a free for all until it's completely re-done, I think it still fits.

What do you think?

Reply:
Yeah, I think I agree on all accounts, and the comment does the job 👍